lib/migrations/10-achievement-2.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.addColumn('achievement', 'flag_name', Sequelize.STRING, {transaction: t}),
+        queryInterface.addColumn('achievement', 'journey_map', Sequelize.STRING, {transaction: t}),
```

Comment:
This will not necessarily be a country flag. So, I have set it as a string token. We will have a corresponding SVG file for the image baked in the app.

Reply:
Renamed journey_map to map_image